### PR TITLE
[MeteoAlerte] Update for OH3

### PR DESCRIPTION
**Pull request by clinique**
Replaced old gif pictures with svg nicer pictures
Added a missing alert report (vague-submersion)
Added end of validity report timestamp
Some code cleansing.

Signed-off-by: clinique <gael@lhopital.org>

### CODE REVIEW

File: bundles/org.openhab.binding.meteoalerte/src/main/java/org/openhab/binding/meteoalerte/internal/handler/MeteoAlerteHandler.java [hunk 1]
```diff
@@ -133,74 +136,55 @@ private void updateAndPublish() {
...
+            if (isLinked(channelId)) {
+                updateState(channelId, new StringType(value.name()));
+            }
+            if (isLinked(channelIcon)) {
+                String resource = getResource(String.format("picto/%s.svg", channelId));
+                if (resource != null) {
+                    resource = resource.replaceAll(ALERT_COLORS.get(AlertLevel.UNKNOWN), ALERT_COLORS.get(value));
+                }
+                updateState(channelIcon,
+                        resource != null ? new RawType(resource.getBytes(), "image/svg+xml") : UnDefType.NULL);
```
Comment (lolodomo) [inline bundles/org.openhab.binding.meteoalerte/src/main/java/org/openhab/binding/meteoalerte/internal/handler/MeteoAlerteHandler.java:176]: Rather UNDER?

**Review by lolodomo — COMMENTED** [review]
Just one remark

File: bundles/org.openhab.binding.meteoalerte/src/main/java/org/openhab/binding/meteoalerte/internal/handler/MeteoAlerteHandler.java (hunk 1 above)
Comment (clinique) [reply bundles/org.openhab.binding.meteoalerte/src/main/java/org/openhab/binding/meteoalerte/internal/handler/MeteoAlerteHandler.java:176]: UNDEF / NULL... I always have this debate :-) I understood that UNDEF was reserved to the core prior any binding update.

Comment (lolodomo) [reply bundles/org.openhab.binding.meteoalerte/src/main/java/org/openhab/binding/meteoalerte/internal/handler/MeteoAlerteHandler.java:176]: That is the reverse. 
NULL is the initial value and should be set only by the framework.

**Comment by clinique** [comment]
With the new glyphs : 
image

**Review by lolodomo — APPROVED** [review]

**Review by cpmeister — APPROVED** [review]
LGTM

**Pull request merged**